packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -249,11 +251,35 @@ function GlobalStylesEditorCanvasContainerLink() {
 			case 'global-styles-css':
 				goTo( '/css' );
 				break;
+			case 'style-book':
+				/*
+				 * The stand-alone style book is open
+				 * and the revisions panel is open,
+				 * close the revisions panel.
+				 * Otherwise keep the style book open while
+				 * browsing global styles panel.
+				 */
+				if ( isRevisionsOpen ) {
+					goTo( '/' );
+				} else {
+					goTo( path );
```

Comment:
This is just a question and not a blocker: does calling `goTo( path );` do anything different to a `return;` statement here? I don't notice any difference in behaviour if I comment out this line or replace it with a `return;` so just thought I'd double check I'm not missing anything. (No need to make any changes to the PR, I'm just curious 🙂)

Reply:
You see... I have this condition where I add or remove random stuff and then forget I did it when everything appears to be working. 

It's never good when playing Jenga.

You're right, I think we can remove it safely.